### PR TITLE
Fix styling for wagtail preview pages 

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@codiebeulaine @miltontony ready for review :)

File: freebasics/templates/polls/question.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
+{% load static core_tags i18n %}
+
+{% block content %}
+
+  <div class="block-poll fb-body fb-card">
+		<h6 class="block-heading fb-accent-1">{% trans "What's your opinion?" %}</h6>
+		<ul>
```
Comment (miltontony) [inline freebasics/templates/polls/question.html]: Why is this `ul` needed

Comment (smarzban) [inline freebasics/templates/polls/question.html]: @miltontony you are right it doesn't need it :)

**Comment by smarzban** [comment]
@miltontony ready for re-review

**Comment by miltontony** [comment]
:+1:

**Pull request merged**